### PR TITLE
REM-1075: Create transactions for open and close node

### PR DESCRIPTION
**Pull request by Art17**
### Jira references
- Ticket (task) — https://remmeio.atlassian.net/browse/REM-1075

### CODE REVIEW

File: remme/tp/node_account.py [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+        if node_account is None:
+            raise InvalidTransaction('Invalid context or address.')
+
+        if node_account.node_state != NodeAccount.NEW:
+            raise InvalidTransaction('Masternode is already opened or closed.')
+
+        if node_account.balance < internal_transfer_payload.value:
+            raise InvalidTransaction('Insufficient amount of tokens on operational account.')
+
+        minimum_stake = int(_get_setting_value(context, SETTINGS_MINIMUM_STAKE))
```
Comment (eugene-babichenko) [inline remme/tp/node_account.py]: https://github.com/Remmeauth/remme-core/pull/233#discussion_r261522347

File: testing/unit/tp/node_account/test_close_masternode.py [hunk 2]
```diff
@@ -0,0 +1,180 @@
...
+UNFROZEN = 10000
+OPERATIONAL = 5000
+
+
+def test_close_masternode():
+    """
+    Case: close masternode.
+    Expect: masternode is closed, all reputation tokens are transferred to operational account.
+    """
+
```
Comment (dmytrostriletskyi) [inline testing/unit/tp/node_account/test_close_masternode.py]: No blank line after function docstring.

File: testing/unit/tp/node_account/test_close_masternode.py [hunk 3]
```diff
@@ -0,0 +1,180 @@
...
+    empty_payload = CloseMasternodePayload()
```
Comment (dmytrostriletskyi) [inline testing/unit/tp/node_account/test_close_masternode.py]: `empty_payload > close_masternode_payload`.

File: testing/unit/tp/node_account/test_close_masternode.py [hunk 4]
```diff
@@ -0,0 +1,180 @@
...
+    NODE_ACCOUNT_FROM_PRIVATE_KEY,
+    INPUTS,
+    OUTPUTS,
+    TRANSACTION_REQUEST_ACCOUNT_HANDLER_PARAMS,
+    create_context,
+)
+
+FROZEN = 300000
+UNFROZEN = 10000
+OPERATIONAL = 5000
```
Comment (dmytrostriletskyi) [inline testing/unit/tp/node_account/test_close_masternode.py]: Clarify constant. Ask yourself `frozen what?`. Frozen balance, frozen amount to transfer to, etc.

File: remme/tp/node_account.py (hunk 1 above)
Comment (Art17) [reply remme/tp/node_account.py]: Done. Check for None and integer.

File: testing/unit/tp/node_account/test_close_masternode.py (hunk 2 above)
Comment (Art17) [reply testing/unit/tp/node_account/test_close_masternode.py]: Blank line removed.

File: testing/unit/tp/node_account/test_close_masternode.py (hunk 3 above)
Comment (Art17) [reply testing/unit/tp/node_account/test_close_masternode.py]: Done.

File: testing/unit/tp/node_account/test_close_masternode.py (hunk 4 above)
Comment (Art17) [reply testing/unit/tp/node_account/test_close_masternode.py]: Done. FROZEN_BALANCE.

**Pull request merged**